transport/http/client.go

```diff
@@ -118,6 +140,13 @@ func (c Client) Endpoint() endpoint.Endpoint {
 	}
 }
 
+// ClientFinalizerFunc can be used to perform work at the end of a client HTTP
+// request, after the response is returned. The principal
+// intended use is for request logging. In addition to the response code
+// provided in the function signature, additional response parameters are
+// provided in the context under keys with the ContextKeyResponse prefix.
+type ClientFinalizerFunc func(ctx context.Context, code int, r *http.Request)
```

Comment:
makes sense to me. the main reason i went with the signature of the ServerFinalizer was from a lack of how to tackle the problem. But it makes sense to me to have a signature like that. I'll update the PR later this evening and push up the requested changes. Thanks for the comments 😸 !